### PR TITLE
Cleaning up and fixing setup.py

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-    "katello.client.cli", 
-    "katello.client.core", 
-    "katello.client.lib", 
+    "katello",
+    "katello.client",
+    "katello.client.api",
+    "katello.client.cli",
+    "katello.client.core",
+    "katello.client.lib",
     "katello.client.lib.ui",
```
Comment (mccun934) [inline setup.py:11]: is this change necessary?

Comment (daviddavis) [inline setup.py:11]: It's not necessary but I figured I'd clean it up while I was in setup.py.

Edit: I don't know why it doesn't show up in github but there's a blank space at the end of each line that I removed. You can highlight the text to view it.

**Comment by mccun934** [comment]
ACK, pending travis. I'm assuming your whitespace changes are just tabs vs spaces

**Comment by daviddavis** [comment]
yea, it's removing trailing whitespace.

**Comment by omaciel** [comment]
Awesome!!!
